coqpp/coqpp_parse.mly

```diff
@@ -391,6 +391,9 @@ rules:
 rule:
 | symbols_opt ARROW CODE
   { { gprod_symbs = $1; gprod_body = $3; } }
+| symbols
```

Comment:
This one is a *cavalier législatif*, and since we have another PR containing it I don't think it should go in this one. (Regardless, I think this feature is making the code *less* robust since you can easily get into silent rule mismatch when rebasing code.)

Reply:
Beside being convenient, this was also to restore a symmetry with the output of `Print Grammar`. Would you prefer we modify the printing to adapt it to coqpp?